templates/htk/fragments/toolbar/base.html

```diff
@@ -11,4 +11,8 @@
 </a>
 {% load htk_tags %}
 {% loadjs '/static/htk/js/toolbar/emulate_user.js' %}
+
+<script type="text/javascript">
```

Comment:
can we rename this file to `toolbar/toolbar.html`

Reply:
also put this above `loadjs` -- the reason this is working is because you have the `$(function() { .... });` wrapper, but otherwise this variable needs to be defined ahead of time